discovery.md

```diff
@@ -0,0 +1,16 @@
+# Box-client discovery: two options
+
+## Option 1: --no-cloud
+
+When the Foxbox is built with the --no-cloud flag, discovery works as follows:
+
+* Connect the box to the router using an ethernet cable. The router can just be a local switch,
+it does not need to be connected to the internet.
+* The box announces its hostname, which has a distinguishable format, e.g. foxbox<i>.local where <i> is a number
```

Comment:
I was thinking - always use number 0 by default, but it that's already taken (the Box can see that by browsing existing mDNS announcements on the LAN), use number 1,2,3, etc. until you find a number that's free.

The app would just discover number 0,1,2,3,.. until it finds no more Boxes on the LAN.